src/foam/nanos/auth/login/LoginView.js

```diff
@@ -0,0 +1,477 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
```

Comment:
This was refactored a while ago and now is way smaller. So the new one should be copied over to here or this should be removed https://github.com/kgrgreer/foam3/blob/development/src/foam/u2/view/LoginView.js

Reply:
Looking at the code here there are a few changes that will need to be carried over. But there is still a lot of extras in this file that are no longer used/required

Reply:
Relevant commits: 
https://github.com/kgrgreer/foam3/commit/9787a3656492088112cce7e7502b8074dd7ca2c9
https://github.com/kgrgreer/foam3/commit/22f10e7c9fcb71c11b0da511ebb0e80edfd7d1a1
https://github.com/kgrgreer/foam3/commit/87c508a2d9a8b3d331bcb5b128450ac50612631a
https://github.com/kgrgreer/foam3/commit/7077215d1e8adf02ee36054d15c50c37dae02443#diff-39db26af7503c6abf62d31fbd2bd5c2dc4b2ca1f47bc8c9822de87995d7cd972
https://github.com/kgrgreer/foam3/commit/564f57265c9b9b38a3e888712b4faf59232d8340#diff-39db26af7503c6abf62d31fbd2bd5c2dc4b2ca1f47bc8c9822de87995d7cd972
https://github.com/kgrgreer/foam3/commit/4db24ea808273be93047255700911330481d3cbf#diff-39db26af7503c6abf62d31fbd2bd5c2dc4b2ca1f47bc8c9822de87995d7cd972
https://github.com/kgrgreer/foam3/commit/2c9834d8c4a576b8c340826b3a459705b16c2f54